packages/common/src/utils/index.ts

```diff
@@ -1,6 +1,26 @@
 import { Validator } from './Validator'
-import { EntityManager } from '@mikro-orm/core'
+import { EntityManager, ReferenceType } from '@mikro-orm/core'
 import { ResourceContract } from '@tensei/core'
+import { FilterOperators } from '@tensei/common'
+
+export const topLevelOperators: FilterOperators[] = ['_and', '_or', '_not']
+export const filterOperators: FilterOperators[] = [
```

Comment:
Please rebase. This has changed on master. You'd have to use the new way of `Utils.graphql. populateFromResolvedNodes()...`